odtp/storage.py

```diff
@@ -17,7 +17,6 @@ def __init__(self):
 
     def test_connection(self):
         bucket = self.s3.head_bucket(Bucket=self.bucketName)
```

Comment:
**suggestion**: why assign the bucket if it is not returned?

Reply:
Accepted